ml-agents/mlagents/trainers/models.py

```diff
@@ -26,9 +29,17 @@ class LearningRateSchedule(Enum):
     LINEAR = "linear"
 
 
-class LearningModel(object):
+class LearningModel:
     _version_number_ = 2
 
+    # Minimum supported side for each encoder type. If refactoring an encoder, please
+    # adjust these also.
+    MIS_RESOLUTION_FOR_ENCODER = {
```

Comment:
nit: should this be `MIN` and not `MIS`?

Reply:
🤦‍♂ yup, will fix